libflux/src/flux/semantic/nodes.rs

```diff
@@ -1603,137 +1606,11 @@ pub fn convert_duration(duration: &Vec<ast::Duration>) -> std::result::Result<Du
 mod tests {
     use super::*;
     use crate::ast;
-    use crate::parser::parse_string;
-    use crate::semantic::analyze::analyze_with;
-    use crate::semantic::import::Importer;
-    use crate::semantic::types::{MonoType, PolyType, Tvar};
+    use crate::semantic::types::{MonoType, Tvar};
     use crate::semantic::walk::{walk, Node};
     use maplit::hashmap;
-    use std::collections::HashMap;
     use std::rc::Rc;
 
-    struct TestImporter {
```

Comment:
I guess all this testing code became redundant with the new code in bootstrap.rs?

Reply:
Yes.